housingbayarea/bloom
# 578/missing translations

# Pull Request Template

## Issue Overview

This PR addresses #578 


## Description

This PR includes translation updates which can be found in the .json files for the general translations, .md files for the resource card translations, and in a script to account for the email translations.

Translations: https://docs.google.com/spreadsheets/d/15OTLhXTtPO-xbwn-1h5dDl3myQaIn_WxymfZu6iedes/edit#gid=1104439968

## How Can This Be Tested/Reviewed?

This can be tested by running yarn db:reseed in backend/core or just yarn db:migration:run if you've previously reseeded in hba. In order to check these translations, sign-in or create an account locally tied to your personal email. Then, fill/autofill applications in each language ensuring that "Do they work in the City of San Jose?" is translated in the process. Also, verify that the email confirmation reflects the translations in the sheet in the issue. 

Finally, sign out, start an application and then leave the tab until the application times out. The updated translations should appear in an alert.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
